openhcl/underhill_core/src/servicing.rs

```diff
@@ -84,6 +84,8 @@ mod state {
         pub dma_manager_state: Option<OpenhclDmaManagerState>,
         #[mesh(10002)]
         pub vmbus_client: Option<vmbus_client::SavedState>,
+        #[mesh(10003)]
+        pub use_vmgs: bool,
```

Comment:
should we use an enum or bool for future compatability, or do we not think we will need anything else?